HoG fountains now give an omnizine-like reagent instead of Doctor's Delight + Typo Fix

PKPenguin321 opened the pull request:
Doctors Delight has piss-poor healing properties since goofball nerfed it to death, and also drains your nutrition unless you're a doctor. This makes it not a very good chem for a structure that's designed to heal you. ~~This PR changes the chemical to omnizine. 20u of omnizine wont overdose and will heal a very decent amount with essentially no drawbacks.~~ This PR adds a new chemical called Godblood that is identical to omnizine except that it has an OD threshold of 150, and sets the fountains to dispense that, instead.

Fixes #13066  (hopefully)

Also fixes a typo in the description for the Nexus.

Fixes #13084

@RemieRichards If you intend to add a check for enemy cultists and need to edit this file, feel free to close this PR and include this change in yours so you don't have to deal with merge conflicts.

:cl: PKPenguin321
tweak: Healing Fountains in Hand of God now give cultists a better and more culty healing chemical instead of Doctor's Delight.
/:cl:

PKPenguin321 committed:
HoG fountains now give omnizine instead of DD

doctors delight has piss-poor healing properties since goofball nerfed it to death, and also drains your nutrition unless you're a doctor. not a very good chem for a fountain designed to heal you. 20u of omnizine wont overdose you and will heal a very decent amount.

PKPenguin321 committed:
fixes typo in nexus description

Fixes #13084

just throwing this in here since it's in the same .dm as the current fix and it's only one line so what could go wrong

Tokiko1 commented:
Consider that omnizine does not instantly heal. Some people may try to drink two pools if they don't see their health fully restored immediatly, which will then result in their death. Since cult has 3 unique reagents for its mode, I am sure you could just make a custom one for the pools to prevent this situation.

optimumtact commented:
It would be fine if overdosing actually gave more feedback, like a status popup so you know you're dying thanks to overdose, instead of just appearing to lose health.

Iamgoofball commented:
@optimumtact you know it does, right?

and than when you OD you get XBOX HUEG RED TEXT telling you what you OD'd on

PKPenguin321 commented:
will update with a new reagent that's just a child of omnizine but with 150 OD threshold soon :tm:

PKPenguin321 committed:
adds godblood, child of omnizine

is identical to omnizine except that it has a 7.5x higher overdose threshold and has a new name/desc.

PKPenguin321 committed:
fountains give godblood instead of omnizine

makes fountains give godblood instead of omnizine to bypass the OD issues with omnizine

PKPenguin321 commented:
in addition to those commits i also actually tested the changes locally to confirm they work. wew

hornygranny commented:
If you heal yourself to full life and then drink to just under the OD limit for godblood, you will be pretty unstoppable in combat

PKPenguin321 commented:
to be fair, that's just a problem with the chems system in general. not to mention it wont help that much in combat if you get outright killed/cuffed, and both are pretty likely since it doesn't touch stun times, just gives light regen. plus if you can get enough fountains at once to actually get 149u of godblood in your system then maybe you kinda deserve it.

Anonus commented:
You'd need literally 7 fountains to hop one person up on godblood, and that's impractical amounts of materials to spend on something that isn't even an objective.

Cheridan commented:
sounds fun
